go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/uber/tchannel-go
 
-go 1.17
+go 1.21
```

Comment:
this should be 1.20 - see https://github.com/uber-go/fx/pull/1111/files

Reply:
ohh yea. Good callout. @zhiyipanuber can you patch this?

Reply:
actually talked to @rabbbit and @zhiyipanuber offline, this should be fine for now as this future code won't build with version lower than 1.20.0.